operators/forklift-operator/2.4.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,10 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: forklift-operator
+  operators.operatorframework.io.bundle.channels.v1: stable-v2.4
+  operators.operatorframework.io.bundle.channel.default.v1: stable-v2.4
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  com.redhat.openshift.versions: "=v4.11"
```

Comment:
@mnecas shouldn't this be `v4.11-v4.12`?

Reply:
It's possible but I used the convention from previous forklift releases.
Probably we should fix all of them.